exec/cnex/lib/process_win32.c

```diff
@@ -61,50 +85,39 @@ void process_call(TExecutor *exec)
     if (!CreateProcess(NULL, command, NULL, NULL, TRUE, 0, NULL, NULL, &si, &pi)) {
         error = GetLastError();
         char szErr[32];
-        snprintf(szErr, 32, "%d", error);
+        snprintf(szErr, sizeof(szErr), "%d", error);
         exec->rtl_raise(exec, "ProcessException", szErr, BID_ZERO);
         free(command);
         return;
     }
     CloseHandle(pi.hThread);
     CloseHandle(out_write);
     CloseHandle(err_write);
-    Cell *err = cell_createStringCell(0); err->type = cBytes;
-    Cell *out = cell_createStringCell(0); out->type = cBytes;
     free(command);
 
-    while (out_read != INVALID_HANDLE_VALUE || err_read != INVALID_HANDLE_VALUE) {
-        char buf[1024];
-        DWORD n;
-        if (out_read != INVALID_HANDLE_VALUE) {
-            if (!ReadFile(out_read, buf, sizeof(buf), &n, NULL)) {
-                CloseHandle(out_read);
-                out_read = INVALID_HANDLE_VALUE;
-                continue;
-            }
-            out->string = string_appendData(out->string, buf, n);
-        }
-        if (err_read != INVALID_HANDLE_VALUE) {
-            if (!ReadFile(err_read, buf, sizeof(buf), &n, NULL)) {
-                CloseHandle(err_read);
-                err_read = INVALID_HANDLE_VALUE;
-                continue;
-            }
-            err->string = string_appendData(err->string, buf, n);
-        }
-    }
-    if (out_read != INVALID_HANDLE_VALUE) {
-        CloseHandle(out_read);
-    }
-    if (err_read != INVALID_HANDLE_VALUE) {
-        CloseHandle(err_read);
-    }
+    struct ThreadInfo stdInfo[2] = { INVALID_HANDLE_VALUE, NULL };
+    stdInfo[0].pHandle = out_read;
+    stdInfo[0].buffer = cell_createStringCell(0); stdInfo[0].buffer->type = cBytes;
```

Comment:
Please put this code on two lines. Statements belong on one line each.

Reply:
Fixed.